README.md

```diff
@@ -18,64 +17,75 @@ Then, add one or more of the modules below:
+@SerializabilityTrait
+trait MySerializable
+
 ```
 
-It may also be worth including additional codecs for common types that are missing in Borer standard library:
+Installation:
 
 ```scala
-libraryDependencies += repo %% "borer-extra-codecs" % commit
+libraryDependencies += "com.github.VirtusLab.akka-serialization-helper" %% "serializability-checker-library" % commit
+libraryDependencies += compilerPlugin("com.github.VirtusLab.akka-serialization-helper" %% "serializability-checker-compiler-plugin" % commit)
 ```
 
-### 2. Checker Plugin
+### 2. Dump Event Schema
 
-A Scala compiler plugin that detects messages, events etc. and checks, whether they extend the base trait and reports
-and error when it doesn't. This ensures that the specified serializer is used by Akka and protects against accidental use
-of Java serialization.
+An sbt plugin that allows for dumping schema
+of [akka-persistence](https://doc.akka.io/docs/akka/current/typed/persistence.html#example-and-core-api) events to a
+file. Can be used for detecting accidental changes of events.
+
+Installation:
 
-To use, just annotate a base trait with `@org.virtuslab.akkasaferserializer.SerializabilityTrait`:
+Add to `plugins.sbt`:
 
 ```scala
-@SerializabilityTrait
-trait MySerializable
+libraryDependencies += "com.github.VirtusLab.akka-serialization-helper" % "sbt-dump-event-schema" % commit
 ```
 
-Installation:
+To enable the plugin for a specific project, use:
 
 ```scala
-libraryDependencies += repo %% "akka-serializability-checker-library" % commit
-libraryDependencies += compilerPlugin(repo %% "akka-serializability-checker-plugin" % commit)
+enablePlugins(DumpEventSchemaPlugin)
 ```
 
-### 3. Dump Schema
+You also have to change one setting responsible for resolving companion compiler plugin:
 
-An sbt plugin that allows for dumping schema of events to a file. Can be used for detecting accidental changes of
-events.
+```scala
+dumpEventSchema / dumpEventSchemaCompilerPlugin := "com.github.VirtusLab.akka-serialization-helper" % "dump-event-schema-compiler-plugin" % commit 
+```
 
-Installation:
+### 3. Serializer
+
+Simple [Borer](https://github.com/sirthias/borer) based Akka serializer. It uses codecs, provided by Borer, that are
+generated during compile time (so serializer won't crash during runtime like reflection-based serializers may do).
 
-Add to `plugins.sbt`:
 ```scala
-libraryDependencies += repo % "sbt-dumpschema" % commit
+libraryDependencies += "com.github.VirtusLab.akka-serialization-helper" %% "borer-akka-serializer" % commit
 ```
-To enable the plugin for a specific project, use:
+
+It may also be worth including additional codecs for common types that are missing in Borer standard library:
+
 ```scala
-enablePlugins(DumpSchemaPlugin)
+libraryDependencies += "com.github.VirtusLab.akka-serialization-helper" %% "borer-extra-codecs" % commit
 ```
 
 ## Comparison with other Akka Serializers
 
 | Serializer | Jackson | Circe | Protobuf v3 | Avro | Borer |
```

Comment:
Pls add a link for each serializer (not just link to the main page of the given library, but rather to the page specifically dedicated to Akka serialization)